packages/loaders/openapi/tests/example_api.test.ts

```diff
@@ -18,12 +17,17 @@ describe('example_api', () => {
     createdSchema = await loadGraphQLSchemaFromOpenAPI('test', {
       fetch,
       baseUrl,
-      oasFilePath: join(__dirname, '../../../handlers/openapi/test/fixtures/example_oas.json'),
+      oasFilePath: '../../../handlers/openapi/test/fixtures/example_oas.json',
+      cwd: __dirname,
     });
     await startServer(PORT);
   });
   afterAll(() => stopServer());
 
+  it('should generate the schema correctly', () => {
+    expect(printSchemaWithDirectives(createdSchema)).toMatchSnapshot();
+  });
+
```

Comment:
@ardatan  This actually already exists (last test in the file)

Reply:
You're right @gilgardosh ! I removed it. You can merge the PR if it  looks good for you.